packages/salesforcedx-vscode-soql/test/vscode-integration/sfdx.test.ts

```diff
@@ -14,16 +20,29 @@ describe('sfdx utils', () => {
     sandbox.restore();
   });
 
-  it('should display an error, channel log, and send telemetry if can not get connection to org', async () => {
-    sandbox.stub(sfdx.workspaceContext, 'getConnection').throws();
-    const vscodeErrorMessageSpy = sandbox.spy(
-      vscode.window,
-      'showErrorMessage'
-    );
-    const channelServiceSpy = sandbox.spy(sfdx.channelService, 'appendLine');
-    await sfdx.withSFConnection(async () => {});
-    sfdx.debouncedShowChannelAndErrorMessage.flush();
-    expect(vscodeErrorMessageSpy.callCount).to.equal(1);
-    expect(channelServiceSpy.callCount).to.equal(1);
+  describe('withSFConnection', () => {
+    function run(showErrorMessage: boolean) {
+      it(`should ${
+        showErrorMessage ? '' : 'not '
```

Comment:
Interesting formatting - was this intentional?

Reply:
This was Prettier's doing. I agree, it is "interesting." Let me see if this is enforced or was just a suggestion.